setup.sh

```diff
@@ -0,0 +1,295 @@
+sed -i -e "s/^define('DB_HOST', '.*')/define('DB_HOST', '$SqlHost')/" $WebFolder/conf.php
+sed -i -e "s/^define('DB_NAME', '.*')/define('DB_NAME', '$SqlDb')/" $WebFolder/conf.php
+
+#apply adjustments for MTAs
+PS3='Which MTA do you want to use with MailWatch?: (it should already be installed):'
+options=("sendmail" "postfix" "exim" "skip")
+select opt in "${options[@]}"
+do
+    logprint "Selected mta $opt"
+    case $opt in
+        "sendmail")
+            logprint "Not yet supported"
+#TODO
+            sleep 1
+            break
+            ;;
+        "postfix")
+            logprint "Configure MailScanner for use with postfix"
+            "$InstallFilesFolder/setup.examples/postfix/mailwatch-postfix.sh" "$Webuser"
+            sleep 1
+            break
+            ;;
+
+        "exim")
+            logprint "Configure MailScanner for use with exim"
+            "$InstallFilesFolder/setup.examples/exim/mailwatch-exim.sh" "$Webuser"
+            sleep 1
+            break
+            ;;
+        *)
+            logprint "Not configuring mta"
+            sleep 1
+            break
+            ;;
+    esac
+done
+
+logprint "Adjusting perl files"
+sed -i -e "s/my (\$db_name) = '.*'/my (\$db_name) = '$SqlDb'/" "$InstallFilesFolder/MailScanner_perl_scripts/MailWatch.pm"
+sed -i -e "s/my (\$db_host) = '.*'/my (\$db_host) = '$SqlHost'/" "$InstallFilesFolder/MailScanner_perl_scripts/MailWatch.pm"
+sed -i -e "s/my (\$db_user) = '.*'/my (\$db_user) = '$SqlUser'/" "$InstallFilesFolder/MailScanner_perl_scripts/MailWatch.pm"
+sed -i -e "s/my (\$db_pass) = '.*'/my (\$db_pass) = '$SqlPwd'/" "$InstallFilesFolder/MailScanner_perl_scripts/MailWatch.pm"
+sed -i -e "s/my (\$db_name) = '.*'/my (\$db_name) = '$SqlDb'/" "$InstallFilesFolder/MailScanner_perl_scripts/SQLSpamSettings.pm"
+sed -i -e "s/my (\$db_host) = '.*'/my (\$db_host) = '$SqlHost'/" "$InstallFilesFolder/MailScanner_perl_scripts/SQLSpamSettings.pm"
+sed -i -e "s/my (\$db_user) = '.*'/my (\$db_user) = '$SqlUser'/" "$InstallFilesFolder/MailScanner_perl_scripts/SQLSpamSettings.pm"
+sed -i -e "s/my (\$db_pass) = '.*'/my (\$db_pass) = '$SqlPwd'/" "$InstallFilesFolder/MailScanner_perl_scripts/SQLSpamSettings.pm"
+sed -i -e "s/my (\$db_name) = '.*'/my (\$db_name) = '$SqlDb'/" "$InstallFilesFolder/MailScanner_perl_scripts/SQLBlackWhiteList.pm"
+sed -i -e "s/my (\$db_host) = '.*'/my (\$db_host) = '$SqlHost'/" "$InstallFilesFolder/MailScanner_perl_scripts/SQLBlackWhiteList.pm"
+sed -i -e "s/my (\$db_user) = '.*'/my (\$db_user) = '$SqlUser'/" "$InstallFilesFolder/MailScanner_perl_scripts/SQLBlackWhiteList.pm"
+sed -i -e "s/my (\$db_pass) = '.*'/my (\$db_pass) = '$SqlPwd'/" "$InstallFilesFolder/MailScanner_perl_scripts/SQLBlackWhiteList.pm"
+
+logprint "Copying perl files to MailScanner"
+cp "$InstallFilesFolder"/MailScanner_perl_scripts/* /etc/MailScanner/custom/
+
+logprint "Restart mailscanner service"
+/etc/init.d/mailscanner restart
```

Comment:
If mailscanner is installed by the script at the end the user gets a message that he has to configure needed settings in mailscanner.conf and the defaults file. And because the user should at least look at mailscanner configuration he should set this himself I think